app/lib/currencies_loader.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CurrenciesLoader
+  DEFAULT_CURRENCIES = {
+    'USD - American Dollar' => 'USD'
```

Comment:
https://github.com/3scale/porta/blob/204d1f00c9885dbcc7bfa21301f427cfe7413005/db/schema.rb#L264

Reply:
A funny one for you
https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/db/schema.rb#L899

Reply:
![This is fine GIF](https://media.giphy.com/media/z9AUvhAEiXOqA/giphy.gif)